Packs/PrismaAccess/Playbooks/playbook-Prisma_Access_-_Connection_Health_Check.yml

```diff
@@ -0,0 +1,384 @@
+          "y": 1320
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: 83778b88-6030-4579-88e3-3614b58e504f
+    type: condition
+    task:
+      id: 83778b88-6030-4579-88e3-3614b58e504f
+      version: -1
+      name: Is Prisma Access Enabled?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#default#':
+      - "7"
+      "yes":
+      - "8"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: modules
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.state
+                      iscontext: true
+                    right:
+                      value:
+                        simple: active
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: modules.brand
+                      iscontext: true
+                    right:
+                      value:
+                        simple: Prisma Access
+                accessor: brand
+            iscontext: true
+          right:
+            value:
+              simple: Prisma Access
+    view: |-
+      {
+        "position": {
+          "x": 607.5,
+          "y": 35
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: fc8bd1c8-ba05-4daa-871d-6336916fb9f0
+    type: title
+    task:
+      id: fc8bd1c8-ba05-4daa-871d-6336916fb9f0
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 890,
+          "y": 1500
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "8":
+    id: "8"
+    taskid: 88992f90-9164-4dc9-8982-e574a28879aa
+    type: title
+    task:
+      id: 88992f90-9164-4dc9-8982-e574a28879aa
+      version: -1
+      name: Query Prisma Access
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+      - "2"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "9":
+    id: "9"
+    taskid: 8bea80c0-59c0-487f-8a6f-05b74f4df71b
+    type: regular
+    task:
+      id: 8bea80c0-59c0-487f-8a6f-05b74f4df71b
+      version: -1
+      name: Remediate tunnel issue
```

Comment:
Adding a generic recommendation (what actions will be taken) in the description is beneficial.

Reply:
Adding description.